### PR TITLE
998717-no-signo - headpin mode skips signo

### PR DESCRIPTION
**Pull request by thomasmckay**
998717-no-signo - made executable

Set sso enable to false

### CODE REVIEW

File: upgrade-scripts/0760_headpin_disable_signo.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+#apply: headpin
+#run: always
+#description:
+#This step forces the single sign-on authentication, Signo, off
+
+sed -i '/  sso:/{
+N
+s/  sso:\n    enable: true/  sso:\n    enable: false/
+}' /etc/katello/katello.yml
+ret_code=$?
```
Comment (lzap) [inline upgrade-scripts/0760_headpin_disable_signo.sh:13]: Nice seding here :-)

**Comment by iNecas** [comment]
ACK

**Comment by lzap** [comment]
From my side ACK

**Comment by jlsherrill** [comment]
would it not make sense to implement this as part of katello-configure which gets run during the katello-upgrade process?  IS there some reason to not do it there?

**Comment by thomasmckay** [comment]
@jlsherrill For headpin currently (and next release of SAM) this is not optional. It is my hope that as the katello/foreman integration gets better that there won't be a need for headpin to behave differently.

**Pull request merged**